sofa-boot-core/src/main/java/com/alipay/sofa/rpc/boot/container/ServerConfigContainer.java

```diff
@@ -420,34 +421,30 @@ ServerConfig createDubboServerConfig() {
      * 释放所有 ServerConfig 对应的资源，并移除所有的 ServerConfig。
      */
     public void closeAllServer() {
-        if (boltServerConfig != null) {
-            boltServerConfig.destroy();
-            boltServerConfig = null;
-        }
-
-        if (restServerConfig != null) {
-            restServerConfig.destroy();
-            restServerConfig = null;
-        }
+        destroyServerConfig(boltServerConfig);
```

Comment:
This modification ignores the logic of `config` setting to null, when `ServerConfigContainer` close and reopen again, `xxServerConfig`\'s initialization logic will be ignored.